uxarray/grid/grid.py

```diff
@@ -1390,3 +1396,39 @@ def isel(self, **dim_kwargs):
             raise ValueError(
                 "Indexing must be along a grid dimension: ('n_node', 'n_edge', 'n_face')"
             )
+
+    def compute_dual(self, method="global"):
```

Comment:
Wonder if `get_dual()` might be more appropriate. Any thoughts?

Reply:
+1: Not a computational operator, so get_ makes more sense.